includes/class-wcs-att-display.php

```diff
@@ -65,16 +67,26 @@ public static function frontend_scripts() {
 			wp_enqueue_script( 'wcsatt-cart' );
 
 			$params = array(
-				'update_cart_option_nonce' => wp_create_nonce( 'wcsatt_update_cart_option' ),
-				'wc_ajax_url'              => WC_AJAX::get_endpoint( "%%endpoint%%" )
+				'i18n_update_cart_sub_error' => __( 'Failed to update your cart. If this issue persists, please re-load the page and try again.', 'woocommerce-subscribe-all-the-things' ),
+				'i18n_subs_load_error'       => __( 'Failed to load matching subscriptions. If this issue persists, please re-load the page and try again.', 'woocommerce-subscribe-all-the-things' ),
```

Comment:
I don't think these refer to the plugin itself, but rather:

`i18n_update_cart_sub_error` = Error updating cart subscription.

You get the point. As far as I remember none of those keys is prefixed because they don't need to (they are in a prefixed JS variable already). We're nitpicking here, but I really appreciate an eye for detail!!! 🙇